python/ray/runtime_env.py

```diff
@@ -76,6 +79,97 @@ def _parse_proto_plugin_runtime_env(
             runtime_env_dict["plugins"][plugin.class_path] = json.loads(plugin.config)
 
 
+@PublicAPI(stability="beta")
+class RuntimeEnvConfig(dict):
+    """This class is used to define a config for runtime environment.
+
+    Config does not participate in the calculation of the runtime
+    environment hash value, which means that the runtime_env is also
+    considered to be the same runtime_env when all fields except
+    config are the same but the config is different.
+
+    Args:
+        setup_timeout_seconds (int): The timeout of runtime environment
```

Comment:
Comment about the default value?

Reply:
setup_timeout_seconds has no default value, but runtime_env_config has, I think we should comment about the default value of config on runtime_env's commentary

Reply:
how do I specify unlimited? should be able to set `-1` for unlimited.